_saas-integrations/darksky/v1/darksky-v1.md

```diff
@@ -0,0 +1,111 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Dark Sky
+permalink: /integrations/saas/darksky
+keywords: darksky, integration, schema, etl darksky, darksky etl, darksky schema
+layout: singer
+# input: false
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "darksky"
+display_name: "Dark Sky"
+
+singer: true 
+tap-name: "darksky"
+repo-url: https://github.com/singer-io/tap-darksky
+
+# this-version: "1.0"
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+status: "Open Beta"
+certified: false
+
+historical: "1 year"
+frequency: "30 minutes"
+tier: "Free"
+status-url: "https://status.darksky.net/"
+
+anchor-scheduling: true
+cron-scheduling: false
```

Comment:
This is supported, and so should be `true`. Let me know if you need a refresher on how to figure out the support for this feature.